packages/api-react/src/hooks/useLocalStorage.ts

```diff
@@ -3,26 +3,32 @@ import EventEmitter from '../utils/EventEmitter';
 
 const eventEmitter = new EventEmitter();
 
-function getValueFromLocalStorage<T>(key: string, defaultValue: T) {
+function getValueFromLocalStorage<T>(
+  key: string,
+  defaultValue?: T
+): T | undefined {
   const item = window.localStorage.getItem(key);
 
   if (item === undefined || item === null) {
-    return defaultValue;
+    return undefined;
```

Comment:
This change appears to be causing an exception when loading WalletsSidebar with the useHiddenWallet hook.

I'm seeing this when selecting a key at launch (auto-login disabled):

![image](https://user-images.githubusercontent.com/339312/193945754-60fc8f26-4152-4f84-b67f-bb292849100a.png)

Reply:
The issue seems to be that hiddenFingerprints is undefined:

![image](https://user-images.githubusercontent.com/339312/193945897-8a95c89c-6264-4183-98f2-564116b9e62c.png)

Reply:
Seems to work if I add:

`
if (!hiddenWalletids) {
  return false;
}
`

Reply:
Hiding a wallet seems to throw an exception as well

Reply:
I fixed typo in my code. Thanks for nice catch